regression/verilog/SVA/named_property1.desc

```diff
@@ -1,7 +1,7 @@
 CORE
 named_property1.sv
 --bound 0
-^\[main\.assert\.1\] always main\.x == 10: PROVED up to bound 0$
+^\[main\.assert\.1\] always main\.x_is_ten: PROVED up to bound 0$
```

Comment:
I would have expected `x_is_ten` to make it into the property identifier, but maybe that's a misconception on my part?

Reply:
Ah, no, there is a some terminology issue here: The identifier is for assertions, not Verilog properties. Verilog properties may or may not end up as an assertion, assumption, cover statement, etc.

Verilog properties do not get checked by default.